posix/posix.c

```diff
@@ -2367,97 +2350,6 @@ int posix_poll(struct pollfd *fds, nfds_t nfds, int timeout_ms)
 	return ready;
 }
 
-#else
-
-int posix_poll(struct pollfd *fds, nfds_t nfds, int timeout_ms)
```

Comment:
what was the purpose of this code?

Reply:
Alternative approach to poll, never tested, without support in device nor fs drivers. I guess I got carried away with removing it, thought, as I should remove bunch of additional code too. I'll restore it for now.